lib/rules/indent.js

```diff
@@ -1385,6 +1373,15 @@ module.exports = {
                 if (astUtils.isSemicolonToken(lastToken)) {
                     offsets.ignoreToken(lastToken);
                 }
+
+                if (options.VariableDeclarator[node.kind] === "first") {
```

Comment:
For performance, it might be better to skip the lines above (starting with `if (node.declarations...` on line 1344) when the offset is `first`, because those lines would have no effect.